setup/pkg/fieldlabs/environment_create.go

```diff
@@ -157,7 +157,24 @@ func (e *EnvironmentManager) Ensure(envs []Environment, labSpecs []LabSpec) erro
 		}
 	}
 
-	envs, err := e.createApps(envs)
+	envs, appsToCreate, appsToDelete, err := e.getAppsDelta(envs)
+	if err != nil {
+		return errors.Wrap(err, "Failed calculating deltas")
```

Comment:
some of the style guide for go has a point about "errors shouldn't being with capital letters or end with punctuation" 

I think internally we have a convention for `errors.Wrap` that the wrapping text should describe what the code was trying to do, in the present tense, e.g. 


```golang
			return nil, errors.Wrapf(err, "get or create app %q", appName)
```

```golang
		return nil, errors.Wrapf(err, "check for existing app")
```

Reply:
this should probably be checked automatically by some style checker, not sure which one would catch the first point about capitalization